app/server/utils/ussd/kenya_ussd_state_machine.py

```diff
@@ -232,6 +247,14 @@ def process_exchange_token_request(self, user_input):
         amount = float(self.session.get_data('exchange_amount'))
         ussd_tasker.exchange_token(self.user, agent, amount)
 
+    def store_transfer_usage(self, user_input):
+        pass
```

Comment:
this still gets triggered in the after of the state machine, but it'd do nothing? how come the storing happens at the processor level and not here?